src/phasorpy/phasor.py

```diff
@@ -990,10 +991,17 @@ def phasor_calibrate(
         Must be measured with the same instrument setting as the phasor
         coordinates to be calibrated.
     frequency : array_like
-        Laser pulse or modulation frequency in MHz.
-        A scalar or one-dimensional sequence.
+        Fundamental laser pulse or modulation frequency in MHz.
     lifetime : array_like
         Lifetime components in ns. Must be scalar or one dimensional.
+    harmonic: int | Sequence[int] | Literal['all'] | str | None = None,
+        Harmonics included in first axis of `real` and `imag` for calibration.
+        If `'all'`, the harmonics in the first axis of phasor coordinates are
+        all considered for calibration.
+        The default is the first harmonic (fundamental frequency).
+    skip_axis : int or sequence of int, optional
+        Axes to be excluded during center calculation. If None, all
+        axes are considered.
```

Comment:
How about:

```
    lifetime : array_like
        Lifetime components in ns. Must be scalar or one-dimensional.
    harmonic : int, sequence of int, or 'all', default: 1
        Harmonics included in `real` and `imag`.
        If an integer, the harmonics at which `real` and `imag` were acquired
        or calculated.
        If a sequence, the harmonics included in the first axis of `real` and
        `imag`.
        If `'all'`, the first axis of `real` and `imag` contains lower
        harmonics.
        The default is the first harmonic (fundamental frequency).
    skip_axis : int or sequence of int, optional
        Axes to be excluded during center calculation. If None, all
        axes are considered, except for the first axis if multiple harmonics
        are specified.
```

Reply:
Excellent! These are much better descriptions and consider cases I didn't specified before.